georiviere/description/models.py

```diff
@@ -143,10 +143,10 @@ class Morphology(AddPropertyBufferMixin, TopologyMixin, TimeStampedModelMixin,
         on_delete=models.SET_NULL, null=True, blank=True,
         related_name='main_flow'
     )
-    secondary_flow = models.ForeignKey(
+    secondary_flows = models.ManyToManyField(
         FlowType, verbose_name=_("Secondary flow"),
-        on_delete=models.SET_NULL, null=True, blank=True,
-        related_name='secondary_flow'
+        blank=True,
+        related_name='flows_on_second'
```

Comment:
Why is this related name not named `morphologies_on_second` as well ?

Reply:
Indeed

Reply:
Fixed !